mypy/renaming.py

```diff
@@ -74,61 +75,49 @@ def visit_mypy_file(self, file_node: MypyFile) -> None:
         This is the main entry point to this class.
         """
         self.clear()
-        self.enter_scope(FILE)
-        self.enter_block()
-
-        for d in file_node.defs:
-            d.accept(self)
-
-        self.leave_block()
-        self.leave_scope()
+        with self.enter_scope(FILE):
+            with self.enter_block():
```

Comment:
```suggestion
        with self.enter_scope(FILE), self.enter_block():
```
Save a level of indentation. This also applies in the next function.

Reply:
Good call, fixed!